nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/hosted/v1/HostedHandlers.java

```diff
@@ -68,59 +56,54 @@
     Headers headers = context.getRequest().getHeaders();
     String method = context.getRequest().getAction();
 
-    if(headers.contains(CLIENT_CHECKSUM) && method != "PUT") {
+    if (headers.contains(CLIENT_CHECKSUM) && method != "PUT") {
```

Comment:
Is it possible to use org.sonatype.nexus.repository.http.HttpMethods.GET here?

Reply:
Yes, it is possible. But this is not my implementation

Reply:
fixed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/97/commits/35de5335f2bc22babed94ccec4989d164aa63098